buildconfig/Jenkins/Conda/build

```diff
@@ -44,6 +44,12 @@ cmake --preset=${CMAKE_PRESET} ${MANTID_DATA_STORE_CMAKE} ${EXTRA_CMAKE_FLAGS} $
 # Run the actual builds for the code, unit tests, and system tests.
 cd $WORKSPACE/build
 
+# if using ninja let ninja control compiling threads
+if [ -f build.ninja ]; then
```

Comment:
I think it would make sense to combine this into an if/else statement with line 36-39?

In this implementation the fact we don't use `ninja` on windows is inferred as we disregard line 38 if using ninja. We may as well make this explicit?

Reply:
I could add a comment to that effect. I like to keep the platform and the tools as separate things in case the tool becomes more platform generic.

This comes from long ago when pre-processor macros in the code assumed clang==osx which made it really hard when people started testing out clang on linux.

Reply:
Pull requests on Windows are using `ninja`, so this would need to be moved to before line 37, otherwise we lose the `--config Release`